### ISSUE TITLE
Item purchase is not the item given in game.

### ISSUE THREAD

**Issue by NewbsUp** [dalaranwow/dalaran-wow#5336]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

**Expected behaviour**: Tell us what should happen instead

**Steps to reproduce the problem**:

1. Purchased Grand staff of Jordan with vote points because it says the item has better stats 160 int,37 sta, 25 crit. However this is not that staff I received.  I received from my research is the normal staff. But thats not the one i want sense it does not have the stats stated on the web page.
2. The web page should be fixed to say the real stats of the staff and I would like reimbursement of 70 vote points please.
3. Go to vote page, look for two handed items (Staffs), find the grand staff of Jordan. Purchase item to character, look at staff difference and be frustrated, get on GitHub to see if this is an issue and how to fix it, Contact Dalaranwow to see if they can swap out with dignified headmasters charge the one that has that stats that the staff of jordan said it had.

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by NewbsUp** [comment]
![headmasters staff](https://user-images.githubusercontent.com/37151958/37104517-2669994a-21ea-11e8-8923-df1dbebcd030.jpg)
![staff of jordan](https://user-images.githubusercontent.com/37151958/37104519-268c60a6-21ea-11e8-87c5-8af8a76805e9.jpg)

**Comment by NewbsUp** [comment]
![in game staff](https://user-images.githubusercontent.com/37151958/37105001-4cf4a9b4-21eb-11e8-9f1d-21e11fbb8d3d.jpg)

**Comment by alexuktc** [comment]
Bro, you purchased an heirloom staff. This means that its stats increase as you level up. The values you see on the website are the stats for level 80. Do you actually believe you can have a staff with 160 intellect and 25 crit rating on level 21? Do you know how fucking overpowered you would be for your level?

**Comment by Rushor** [comment]
-- GM ONLY --
The issuetracker is for serverside and gamemechanic issues only. Please write an Ingame-issue to a Gamemaster and the Gamemaster will then help you as soon as he can. (Or create a new Ingame-Account in order to be able to talk with a gamemaster with you new level 1 character) Thank you.

![GitHub Logo](http://i.imgur.com/bYkYeGQ.png)
